scripts/update_version.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import argparse
+from collections import namedtuple
+
+
+def get_arg():
+    parser = argparse.ArgumentParser(
+        description='Apply the specified version to affected source files')
+    parser.add_argument('version', metavar='version', type=str,
+                        help='The version to apply')
+
+    return parser.parse_args()
+
+
+class Version:
+    RE_PATTERN = r"(\d+)\.(\d+)\.(\d+)(-dev)?"
```

Comment:
Do not limit it to `-dev`.
```suggestion
    RE_PATTERN = r"(\d+)\.(\d+)\.(\d+)(-a-zA-Z0-9)?"
```

Reply:
updated